lib/pages/home/create_movie/widgets/create_movie_button.dart

```diff
@@ -165,7 +165,7 @@ class _CreateMovieButtonState extends State<CreateMovieButton> {
             // Make sure it is 1080p, h264
             // Also set the framerate to 30 and copy all the streams
             await executeFFmpeg(
-                    '-i $currentVideo -vf "scale=1920:1080" -r 30 -map 0 -c copy -c:v libx264 -crf 18 $tempVideo -y')
+                    '-i $currentVideo -vf "scale=1920:1080" -r 30 -map 0 -c:v libx264 -r 30 -c:a copy $tempVideo -y')
```

Comment:
Since you removed `-c copy` and added `-c:a copy`, you also need to copy subtitles stream with: `-c:s copy`

Reply:
And this does reeconding because the scale, so we need to keep `-crf 18` to preserve quality

Reply:
Done 🙏🏽